sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/NonAsyncTaskShouldNotReturnNull.cs

```diff
@@ -28,6 +28,21 @@ public Task GetTask2()
         }
 
         public Task Foo { get; set; }
+
+        public Task<object> GetFooAsync()
+        {
+            return Task.Run(() =>
+            {
+                if (false)
+                {
+                    return new object();
+                }
+                else
+                {
+                    return null;
+                }
+            });
+        }
```

Comment:
There are two types of ignored enclosing syntax: please add a test for each

Also, please mark those test cases as _// False negative_

Reply:
This test case is not an FP.

Reply:
@Evangelink your comment on the original pull request says that this is only a partial fix which will still produce some FNs (and I assumed that this lambda case was an example of an FN).
Either way, if there are known FNs that will still occur, could you call them out in test cases please?